queries/rust/highlights.scm

```diff
@@ -220,20 +220,11 @@
   (block_comment)
 ] @comment @spell
 
-((line_comment) @comment.documentation
-  (#lua-match? @comment.documentation "^///[^/]"))
+(line_comment
+  (doc_comment)) @comment.documentation
```

Comment:
comment.documentation should only be captured for doc_comment I think

Reply:
Hmm... Won't that mean you highlight the comment markers and the comments differently?

Reply:
Yes.
I personally will do it like that. Unsure if that's the better behavior.

Reply:
I have no skin in this game, so defer to the majority opinion. How do other languages handle that? (Consistency wins.)

Reply:
I'm fine with either; I just want to hear the opinions before merging it to avoid later "this bad we need to change" PRs ;)

Reply:
I think it should be the entire `line_comment`/`block_comment` node for consistency

Reply:
I don't mind either way.

Reply:
I'll go with @amaanq here then.

Reply:
Then I'll go with the Rust maintainer's opinion. (This can always be revisited.)